thinkhazard/templates/faq.jinja2

```diff
@@ -4,7 +4,7 @@
       gettext('General'), [
         (
           gettext('What is the purpose of this tool?'),
-          gettext('*ThinkHazard!* was designed to increase awareness of natural hazards anywhere in the world. It does so by providing a simple overview of different hazards and their ‘hazard levels’ for any given area, delivering recommendations and information to aid users in managing their risk to those hazards. The tool is designed to become increasingly comprehensive over time as users contribute new data and information. If you have data or information to contribute, please [click here](%(link)s).', link='https://goo.gl/forms/FX8uxUgLwDKErdFG2')
+          gettext('*ThinkHazard!* was designed to increase awareness of natural hazards anywhere in the world. It does so by providing a simple overview of different hazards and their ‘hazard levels’ for any given area, delivering recommendations and information to aid users in managing their risk to those hazards. The tool is designed to become increasingly comprehensive over time as users contribute new data and information. If you have data or information to contribute, please [click here](%(link)s). *ThinkHazard!* does not provide tools for detailed risk analysis. [IDF CatRiskTools](%(link2)s) provides a catalog of open access and commercially licenced risk analysis tools. Impact analysis can be conducted on data stored at [GFDRR Innovation Lab GeoNode](%(link3)s) using the new GeoSafe impact analysis tool.', link='https://goo.gl/forms/FX8uxUgLwDKErdFG2', link2='https://catrisktools.oasishub.co',link3='url_to_geonode')
```

Comment:
`url_to_geonode`(link3) looks like a wrong one.

Reply:
This should direct to http://geonode-gfdrrlab.org/. 
It has been defined somewhere in the code I believe, as the code 'url_to_geonode' is being used already elsewhere in FAQ or about pages.